packages/core/src/codewhispererChat/tools/toolShared.ts

```diff
@@ -15,3 +19,16 @@ export interface InvokeOutput {
         content: string
     }
 }
+
+export function sanitizePath(inputPath: string): string {
```

Comment:
DId you look at the existing fs utils here?: https://github.com/aws/aws-toolkit-vscode/blob/ce01fa85c7a97c71cd3dcc11770f6cb75b29e1c2/packages/core/src/shared/utilities/pathUtils.ts#L92

Reply:
There is also `sanitizeFilename`.